examples/sample-app/src/main.js

```diff
@@ -182,7 +182,15 @@ const questions1 = [
     },
     name: "mood",
     message: "Select mood",
-    choices: ["happy", "sad"],
+    choices: [
+      {
+        value: "happy",
+        disabled: true,
+      },
+      {
+        value: "sad",
+      },
+    ],
```

Comment:
this is breaking change !!! all previously created generators will stop working. IMHO, we need to support for both  strings and object values.